code/modules/telescience/radiostation.dm

```diff
@@ -241,17 +241,34 @@
 	icon_state = "mixtable-3"
 	anchored = 1
 	density = 1
-	var/has_record = 0
-	var/is_playing = 0
+	var/can_play_music = TRUE
+	var/has_record = FALSE
+	var/is_playing = FALSE
 	var/obj/item/record/record_inside = null
 
 	New()
-		..()
+		. = ..()
 		MAKE_SENDER_RADIO_PACKET_COMPONENT("pda", FREQ_PDA)
+		START_TRACKING
+
+	get_desc()
+		if(!src.can_play_music)
+			. += " There's an \"out of order\" label on it."
+
+	disposing()
+		STOP_TRACKING
+		. = ..()
+
+	get_desc()
+		if(!src.can_play_music)
+			. += " There's an \"out of order\" label on it."
```

Comment:
?
```suggestion
```

Reply:
Another left-over from when I tried to fix rebasing issues. I'll remove it.